src/Tribe/Commerce/PayPal/Main.php

```diff
@@ -1141,7 +1141,7 @@ public function save_ticket( $post_id, $ticket, $raw_data = array() ) {
 		// Makes sure it's an Int after this point
 		$data['stock'] = (int) $data['stock'];
 
-		if ( '' !== $data['mode'] ) {
+		if ( isset( $data['mode'] ) && '' !== $data['mode'] ) {
```

Comment:
Suggestion: If `$data['mode']` can't be 0 or `false`, you could just use `empty()` here?

If it can ignore the suggestion. This looks 👍

Reply:
I'm not really sure that `0` or `false` would not be legit values... I had the same thought though.